src/ParsePushAdapter.js

```diff
@@ -0,0 +1,153 @@
+"use strict";
+// ParsePushAdapter is the default implementation of
+// PushAdapter, it uses GCM for android push and APNS
+// for ios push.
+
+const Parse = require('parse/node').Parse;
+const GCM = require('./GCM');
+const APNS = require('./APNS');
+
+function ParsePushAdapter() {
+ this.validPushTypes = ['ios', 'android'];
+ this.senders = {};
+}
+
+/**
+ * Register push senders
+ * @param {Object} pushConfig The push configuration which is given when parse server is initialized
+ */
+ParsePushAdapter.prototype.registerPushSenders = function(pushConfig) {
```

Comment:
can you add a dissociation per App ID? I'm almost done for the multiple apps support.